libraries/utils.js

```diff
@@ -1,5 +1,17 @@
+const fs = require('fs')
+
+// picks random number between min and max
 module.exports.random = (min, max) => {
   min = Math.ceil(min);
   max = Math.floor(max);
   return Math.floor(Math.random() * (max - min)) + min
+}
+
+// loads json file from filepath then executes callback
+module.exports.loadJSON = (filepath, callback) => {
```

Comment:
@rashidarmand this is where the sauce is right here.

Reply:
Talking out loud here -> I may have to change this in the future due to me possibly hiding the error here. 🤔